mscore/data/shortcuts.xml

```diff
@@ -982,6 +982,14 @@
     <key>toggle-insert-mode</key>
     <seq>Ctrl+I</seq>
     </SC>
+  <SC>
```

Comment:
After adding a new commit to this branch that restores the previous shortcuts.xml:

<code>git rebase -i HEAD~2</code>

In the vi editor, change the second line to start with <code>f</code> instead of <code>pick</code>, write and quit.

Then <code>git push -f</code> to update this pull request.